src/github.com/getlantern/flashlight/main/test.yaml

```diff
@@ -1,7 +1,7 @@
 version: test
```

Comment:
With the `map` implemntation, the key names become camelCased in YAML. It will invalidate users' existing `settings.yaml`, but it's good to keep the names in same format from YAML file to UI.